README.md

```diff
@@ -1,3 +1,89 @@
 # Accessible Tabs
 
-[...]
+[![Build Status](https://travis-ci.com/grrr-amsterdam/cookie-consent.svg?branch=master)](https://travis-ci.com/grrr-amsterdam/cookie-consent)
+
+### JavaScript utility library
+
+- No dependencies
+- Lightweight (1.1kB minified and gzipped)
+- Spec-compliant semantics, based on [Tabbed Interfaces](https://inclusive-components.design/tabbed-interfaces/) by [ Heydon Pickering](https://twitter.com/heydonworks)
+
+Built with ❤️ by [GRRR](https://grrr.tech).
+
+## Installation
+
+```sh
+$ npm install @grrr/accessible-tabs
+```
+
+Note: depending on your setup [additional configuration might be needed](https://github.com/grrr-amsterdam/cookie-consent/wiki/Usage-with-build-tools). This package is published with untranspiled JavaScript, as EcmaScript Modules (ESM).
+
+## Usage
+
+Import the module and initialize it:
+
+```js
+import Tabs from '@grrr/accessible-tabs';
+
+// Construct and initialize the module with the list to enhance (see 'Markup' section below).
+const tabs = new Tabs(document.querySelector('.js-tablist')); 
```

Comment:
Yeah, I don't know, in my mind almost all our enhancers could also be web components.

```
<accessible-tabs>
  <ul>
    ...
  </ul>
  ... etc ...
</accessible-tabs>
```

This would be almost the same setup as an `enhancer` attribute, with the same fallback HTML in case Javascript fails and stuff like that. 

But anyway, this is off-track indeed and let's continue with the component as-is. It's nice to consider for future work. 
Browser support is of course potentially an issue.